plugins/woocommerce/client/admin/client/marketing/components/PluginCardBody/SmartPluginCardBody.tsx

```diff
@@ -76,7 +85,12 @@ export const SmartPluginCardBody = ( {
 	const renderButton = () => {
 		const buttonDisabled = !! currentPlugin;
 
-		if ( isPluginInstalledNotActivated( plugin.product ) ) {
+		/**
+		 * When the plugin is installed but not activated yet.
+		 */
+		const isPluginInstalledNotActivated = installState === 'installed';
+
+		if ( isPluginInstalledNotActivated ) {
```

Comment:
I think we can avoid creating a const here:

```suggestion
		if ( installState === 'installed' ) {
```

Reply:
If we use `if ( installState === 'installed' )`, it may not be immediately clear to developers that it means "installed but not yet activated". Developers may think that "activated plugins are installed, so using `installState === 'installed'` would also include activated plugins", but that's not the case. 

That's the reason why I came up with the constant with a code comment, to make it clearer.

What do you think? Does this make sense?

Reply:
yes